DataPackages/Goal/1/datapackage.json

```diff
@@ -1,23 +1,26 @@
  {
-                    "name": "Goal 1",
+                    "name": "goal-1",
                     "title": "End poverty in all its forms everywhere",
                     "description": "Goal 1 calls for an end to poverty in all its manifestations, including extreme poverty, over the next 15 years. All people everywhere, including the poorest and most vulnerable, should enjoy a basic standard of living and social protection benefits.",
                     "homepage": "https://unstats.un.org/sdgs",
                     "resources": [{
                             "name": "data",
                             "path": "data.csv",
+                            "encoding": "ISO-8859-1",
```

Comment:
https://goodtables.io/github/roll/SDG/jobs/1

Clicking on `source error` we can see `'utf-8' codec can't decode byte 0xf4 in position 3249: invalid continuation byte`. So I have changed the encoding to `ISO-8859-1` because data packages use `UTF-8` by default.